r2/r2/controllers/ipn.py

```diff
@@ -590,9 +590,9 @@ def finish(self, parameters, txn_id,
                         "supports the site and makes future development "
                         "possible. For example, one month of reddit gold "
                         "pays for 5 instance hours of reddit's servers.")
-            message += "\n\n" + strings.gold_benefits_msg
+            message += "\n\n" + _(strings.gold_benefits_msg)
```

Comment:
@Pokechu22 HAA HAA. I know *what* it is. I don't know why someone named 'morris ganis' made the note nor how it decided to stop existing within the 3 seconds it took me to press "respond on Github"